packages/web/src/components/ai-generated-button/AiGeneratedCallout.module.css

```diff
@@ -49,33 +49,3 @@
 .iconRobot path {
   fill: var(--static-white);
 }
-
-.button {
```

Comment:
love how many custom styles we can remove. We haven't explicitly discussed this, but im down to remove custom styles entirely, like we should just be using standard button styles even if that changes the look and feel of some of our buttons

Reply:
yeah 100%, i think that would be too much chaos for the design team rn but we can do a dedicated pass in the future